### ISSUE TITLE
Add Show/highlight to Logging tab

### ISSUE THREAD

**Issue by mtho11** [kiali/kiali#2831]
**Is your feature request related to a problem? Please describe.**
We now have hide log entries because that was easier to initially do. Now we need show/highlight log entries.

**Describe the solution you'd like**
We need to at least show log entries that match a string (or regex).

Epic: https://github.com/kiali/kiali/issues/2232

**Comment by lwrigh** [comment]
The UXD team will look further into what Openshift is doing to give clearer feedback on the sort of show/hide behavior that should be featured on the logging page.

**Comment by jshaughn** [comment]
@lwrigh We won't block on this, for logs Find/Hide our approach will be to apply Hide, then Find. We will also use that terminology.   That gives us consistency with Graph Find/Hide behavior and terminology.

Let us know whether this is consistent with, not consistent with, or simply unrelated to anything done in console.

**Comment by lwrigh** [comment]
Here are two different options for approaching the toolbar for logging. 

This first option still features the terminology of show/hide. The layout of the fields has been updated and the toggle options have been moved to the second line on the left side. The info tip next to the 'Activate Regex' toggle would provide the user with more info about what Regex is and what it does.
![Logging- Show-Hide](https://user-images.githubusercontent.com/20227374/83427539-e491c300-a3fe-11ea-90f5-5a3fccc6510a.png)

This second options features a similar layout but features one find field as an alternative to having two fields (show/hide) with the goal of simplifying that interaction. 
![Logging- Find](https://user-images.githubusercontent.com/20227374/83427534-e2c7ff80-a3fe-11ea-8a71-c2939b47b9e7.png)

**Comment by jshaughn** [comment]
@lwrigh Thanks for the mocks.  I prefer option 1 because I'm not exactly sure how a single input will handle both find and hide, because both could be applied simultaneously.

I like the first line as-is.   The groupings on the second line look nice visually but functionally it may be strange because the regex switch applies to the Show/Hide expression.  The orientation switch is unrelated to the regex switch.

**Comment by lwrigh** [comment]
Here's an update to the first one so the Regex switch is located closer to the show/hide fields. 
![Logging- Show-Hide](https://user-images.githubusercontent.com/20227374/83440722-6a1f6e00-a413-11ea-8132-314eb63acb86.png)

**Comment by lwrigh** [comment]
I also did an exploration that aligned the logging tab more with some of the Openshift log layouts. Here's it for thought. 
![Logging- More aligned with Openshift](https://user-images.githubusercontent.com/20227374/83440993-d13d2280-a413-11ea-9ed8-b7b48fc43b92.png)

**Comment by mtho11** [comment]
Great! Thanks @lwrigh !

**Comment by lwrigh** [comment]
No problem!

**Comment by jshaughn** [comment]
@lwrigh Yes, I think that's better to have the regex switch located with the Show/Hide inputs.

**Comment by beaumorley** [comment]
One more comment. We talked about changing Show/Hide to Find/Filter. This is the standard in the Dev Console. Do those labels work with the functionality of the controls?

I actually brought up the Logs Find/Filter in the last PatternFly meeting. They where looking for use cases where browser search does not work.

**Comment by jshaughn** [comment]
@beaumorley  What is currently "Show" takes a pattern and displays only log entries containing the pattern.   What is currently "Hide" takes a pattern and displays only log entries not containing the pattern.  If used together only entries containing the "Show" expression and not containing the "Hide" expression will be dispayed.

I'm not sure "Find" works here because we don't highlight occurrences of the pattern.  That is deferred to the browser "Find" (ctrl-f).  Filters tend to specify what you want to see, so maybe what is currently "Show" could be changed tp "Filter".  I don't see a substitute for "Hide".  What do you think?
